tools/internal_ci/linux/grpc_interop_badserver_java.sh

```diff
@@ -36,6 +36,7 @@ export LANG=en_US.UTF-8
 cd $(dirname $0)/../../..
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
+source tools/internal_ci/helper_scripts/prepare_build_interop_rc
 
 tools/run_tests/run_interop_tests.py -l java --use_docker --http2_server_interop $@
```

Comment:
why are  badserver_java and badserver_python separate? it's probably fine for now but we'll want to stop job proliferation.